src/centrality/pagerank.jl

```diff
@@ -12,34 +12,40 @@ is not reached within `n` iterations.
 """
 function pagerank end
 
-@traitfn function pagerank(g::::IsDirected, α=0.85, n=100::Integer, ϵ=1.0e-6)
-    A = adjacency_matrix(g, Float64; dir=:in)
-    S = vec(sum(A, 1))
-    S = 1 ./ S
-    S[find(S .== Inf)] = 0.0
-    M = A'  # need a separate line due to bug #17456 in julia
-    # scaling the adjmat to stochastic adjacency matrix
-    M = (Diagonal(S) * M)'
+@traitfn function pagerank(g::AbstractGraph, α=0.85, n=100::Integer, ϵ=1.0e-6)
```

Comment:
Is this an enhancement to pagerank to accept undirected graphs, or should we still restrict to DiGraphs?

Reply:
The algorithm works for undirected graphs, so I don't see a good reason for restricting it.